Sming/Makefile-project.mk

```diff
@@ -188,7 +193,11 @@ endif
 
 LIBLWIP = lwip
 ifeq ($(ENABLE_CUSTOM_LWIP), 1)
```

Comment:
@d-a-v We can read the value of the `ENABLE_CUSTOM_LWIP` and if it is 1 then we will use the esp-open-lwip version 1. If it is 2 we will use the lwip2 code. This way we don't have to introduce yet another directive.